workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -82,52 +71,33 @@ public static void Build()
             }
         }
 
-        internal static WorkerPlatform[] GetWorkerPlatforms(string workerTypesArg)
-        {
-            WorkerPlatform[] wantedWorkerPlatforms;
-
-            try
-            {
-                wantedWorkerPlatforms = workerTypesArg.Split(',')
-                    .Select(workerType => new WorkerPlatform(workerType.Trim()))
-                    .Distinct()
-                    .ToArray();
-
-                Array.Sort(wantedWorkerPlatforms);
-            }
-            catch (ArgumentException innerException)
-            {
-                throw new ArgumentException(string.Format("Invalid argument: +{0} {1}\n{2}",
-                    BuildConfigNames.BuildWorkerTypes, workerTypesArg,
-                    innerException.Message));
-            }
-
-            return wantedWorkerPlatforms;
-        }
-
-        public static void BuildWorkerForEnvironment(WorkerPlatform workerPlatform, BuildEnvironment targetEnvironment)
+        public static void BuildWorkerForEnvironment(string workerType, BuildEnvironment targetEnvironment)
         {
-            var spatialOSBuildConfiguration = GetBuildConfiguration();
-            var environmentConfig =
-                spatialOSBuildConfiguration.GetEnvironmentConfigForWorker(workerPlatform, targetEnvironment);
+            var spatialOSBuildConfiguration = SpatialOSBuildConfiguration.instance;
+            var environmentConfig = spatialOSBuildConfiguration.GetEnvironmentConfigForWorker(workerType, targetEnvironment);
             var buildPlatforms = environmentConfig.BuildPlatforms;
             var buildOptions = environmentConfig.BuildOptions;
-
-            PathUtil.EnsureDirectoryExists(PlayerBuildDirectory);
+            
+            if (!Directory.Exists(PlayerBuildDirectory))
+            {
+                Directory.CreateDirectory(PlayerBuildDirectory);
+            }
 
             foreach (var unityBuildTarget in GetUnityBuildTargets(buildPlatforms))
             {
-                BuildWorkerForTarget(workerPlatform, unityBuildTarget, buildOptions, targetEnvironment);
+                BuildWorkerForTarget(workerType, unityBuildTarget, buildOptions, targetEnvironment);
             }
         }
+        
+        public static void Clean()
+        {
+            Directory.Delete(PlayerBuildDirectory);
```

Comment:
Doesn't this nuke every other worker as well?

Reply:
yes, we only have a clean function for: "clean all workers" not for "clean a specific worker"